src/main/java/io/vlingo/symbio/store/object/jdbc/jpa/JPAObjectStoreActor.java

```diff
@@ -7,47 +7,139 @@
 package io.vlingo.symbio.store.object.jdbc.jpa;
 
 import io.vlingo.actors.Actor;
+import io.vlingo.actors.Definition;
+import io.vlingo.actors.Logger;
+import io.vlingo.common.Failure;
+import io.vlingo.common.Success;
+import io.vlingo.common.identity.IdentityGenerator;
+import io.vlingo.symbio.Entry;
+import io.vlingo.symbio.EntryAdapterProvider;
 import io.vlingo.symbio.Metadata;
 import io.vlingo.symbio.Source;
+import io.vlingo.symbio.State;
+import io.vlingo.symbio.StateAdapterProvider;
+import io.vlingo.symbio.store.Result;
+import io.vlingo.symbio.store.StorageException;
+import io.vlingo.symbio.store.dispatch.Dispatchable;
+import io.vlingo.symbio.store.dispatch.Dispatcher;
+import io.vlingo.symbio.store.dispatch.DispatcherControl;
+import io.vlingo.symbio.store.dispatch.control.DispatcherControlActor;
 import io.vlingo.symbio.store.object.PersistentObject;
 import io.vlingo.symbio.store.object.PersistentObjectMapper;
 import io.vlingo.symbio.store.object.QueryExpression;
 
+import java.time.LocalDateTime;
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
 
 /**
  * JPAObjectStoreActor
  */
 public class JPAObjectStoreActor extends Actor implements JPAObjectStore {
+  private final DispatcherControl dispatcherControl;
+  private final Dispatcher<Dispatchable<Entry<String>, State<String>>> dispatcher;
```

Comment:
If text only, consider making consistent with `JdbiObjectStoreDelegate` by using `TextEntry` (assuming that compiles here).

Reply:
Now consistent with `JdbiObjectStoreDelegate` that also used  `Entry<String>`